src/main/java/kfusion/java/pipeline/AbstractPipeline.java

```diff
@@ -434,13 +443,31 @@ public void configure(final Device device) {
         /**
          * configure raycast (render model)
          */
-        preTrans = new FloatSE3(new Float6(0, 0, -volumeDims.getX(), 0, 0, 0)).toMatrix4();
-        Float6 value = new Float6(.5f, .5f, .5f, 0, 0, 0);
-        value = Float6.scale(value, volumeDims.getX());
+        FloatArray float6 = new FloatArray(6);
+        float6.set(0, 0);
+        float6.set(1, 0);
+        float6.set(2, -volumeDims.getX());
+        float6.set(3, 0);
+        float6.set(4, 0);
+        float6.set(5, 0);
+
+        preTrans = new FloatSE3(float6).toMatrix4();
+        //Float6 value = new Float6(.5f, .5f, .5f, 0, 0, 0);
+        FloatArray value = new FloatArray(6);
+        value.set(0, 5.0f);
```

Comment:
values need to be -> `        //Float6 value = new Float6(.5f, .5f, .5f, 0, 0, 0);`